app/src/main/java/ch/epfl/sweng/erpa/services/dummy/database/DummyDataService.java

```diff
@@ -82,39 +91,24 @@ public void saveOne(T t) {
             } else if (gameFile.isDirectory()) {
                 throw new IllegalStateException("Trying to write to existing folder, as file! " + gameFile.getAbsolutePath());
             }
-            FileOutputStream fOut = new FileOutputStream(gameFile, false);
-            OutputStreamWriter writer = new OutputStreamWriter(fOut);
-
-            new Yaml().dump(t, writer);
-        } catch (FileNotFoundException ignored) { //we just created the file. it cannot possibly not exist (unless createNewFile threw an error)
+            try (OutputStreamWriter writer = new OutputStreamWriter(new FileOutputStream(gameFile, false))) {
+                new Yaml().dump(t, writer);
+            }
+        } catch (FileNotFoundException ignored) {
+            // We just created the file. it cannot possibly not exist (unless createNewFile threw an error)
```

Comment:
You forgot the uppercase at the beginning of the second sentence. (And the second sentence doesn't sound very english..) (--> It's impossible that the file doesn't exist ?)